test/app/models/Interactive.spec.js

```diff
@@ -0,0 +1,269 @@
+/* eslint-env jasmine */
+import {
+  elementOrderingSolutionSchema,
+  singleSolutionSchema,
+  classificationSolutionSchema
+} from '../../../app/schemas/models/interactives/common/solutions.schema'
+import {
+  interactiveDraggableOrderingSchema,
+  interactiveInsertCodeSchema,
+  interactiveDraggableClassificationSchema,
+  interactiveMultipleChoiceSchema,
+  interactiveFillInCodeSchema,
+  interactiveDraggableStatementCompletionSchema
+} from '../../../app/schemas/models/interactives/common/interactive_types.schema'
+import interactiveSchema from '../../../app/schemas/models/interactives/interactive.schema'
+
+import Ajv from 'ajv'
+
+// If we want to use both draft-04 and draft-06/07 schemas then use { schemaId: 'auto' }
+const ajv = new Ajv()
+
+function schemaCompileTest (schemaObject) {
```

Comment:
I like these tests being extracted and unified! As we move towards `ajv` for more of our schemas in the future, this could be part of our utils and part of how we build up our tests.

Reply:
I agree 👍 👍